src/client/modules/my/dataUpload/dataUpload.ts

```diff
@@ -74,17 +86,30 @@ export default class DataUploader extends LightningElement {
                 headers: {
                     'Content-Type': 'application/json'
                 },
-                body: JSON.stringify([data]) // Remove brackets for single data insertion
+                body: JSON.stringify(dataList)
             }).then((response) => {
                 //check response from server
                 if (response.status == 200) location.reload();
```

Comment:
Can you change this to put location.reload on another line, so that it matches the style of the rest of this file?